sys/usb/usbus/cdc/acm/cdc_acm.c

```diff
@@ -273,13 +293,16 @@ static void _handle_in(usbus_cdcacm_device_t *cdcacm,
         (cdcacm->state != USBUS_CDC_ACM_LINE_STATE_DTE)) {
         return;
     }
+    // copy at most USBUS_CDC_ACM_BULK_EP_SIZE chars from input into ep->buf
```

Comment:
Sorry to bother you with this, but as per style guide this must be a C-style comment:
```Suggestion
    /* copy at most USBUS_CDC_ACM_BULK_EP_SIZE chars from input into ep->buf */
```

Reply:
Right, my fault.